vars/parseStageInfo.groovy

```diff
@@ -266,28 +281,22 @@ void call(Map config = [:]) {
       tag = tag.trim()
     }
 
-    // Highest repeat ftest argument priority is TestRepeat parameter
-    if (params.TestRepeat && params.TestRepeat != '') {
-      ftest_arg_repeat = params.TestRepeat
-    } else {
-      // Next highest priority is a stage specific Test-repeat-* then the general Test-repeat
-      String repeat = cachedCommitPragma(
-        'Test-repeat' + result['pragma_suffix'], cachedCommitPragma('Test-repeat', null))
-      if (repeat) {
-        ftest_arg_repeat = repeat
-      }
+    // Get the ftest --nvme argument from either the build parameters or commit pragmas
+    String ftest_arg = get_ftest_arg(params.TestNvme, 'Test-nvme', result['pragma_suffix'])
+    if (ftest_arg) {
+      ftest_arg_nvme = ftest_arg
     }
```

Comment:
```suggestion
    ftest_arg_nvme = get_ftest_arg(params.TestNvme, 'Test-nvme', result['pragma_suffix'])
```

But per the question about whether the one-liner in `get_ftest_arg()` deserves wrapping in a function, you could just replace the above `get_ftest_arg()` with the Elvis operator construct that it factors down into.  Same could be done for the other uses below here of course.